gulpfile.js/extras/craft/README.md

```diff
@@ -0,0 +1,25 @@
+# Gulp Starter on Craft
+
+## Gulp Rev Craft Plugin
+Include the proper gulp-revisioned version of an asset in your Twig templates with this plugin.
+
+###Usage:
+
+```twig
+<link rel="stylesheet" href="{{ 'assets/stylesheets/app.css' | gulp_rev }}">
+```
+
+
+The output of the Twig filter defaults to the given string/path unless `rev-manifest.json` exists. When `rev-manifest.json` exists, then the string is replaced with the revisioned asset name stored in that file.
+
+
+###Important Notes:
+* The plugin assumes that your `rev-manifest.json` file is in the public folder.
+* Make sure `rev-manifest.json` does not exist when running `npm run gulp` (the development task). This will cause an error since the revisioned file will not exist.
```

Comment:
@mzlock I'm not sure I follow this bit. What happens here when in development mode and the file doesn't exist? It shouldn't error... the filenames should just remain as is and not get replaced (since there is nothing to read and replace with)